app/release_metadata.json

```diff
@@ -1,6 +1,6 @@
 {
-  "version": "3.1.2-rc",
-  "stability": "rc",
+  "version": "3.1.2",
+  "stability": "stable",
```

Comment:
Wait, why this change?

Reply:
@hluchas please remove this change as this is blocking the PR.

Reply:
Hm, it comes from one of my commits, I don't know why. It should not be brought by a PR. I think we can just revert this commit. I think it came upper when we did some maintenance on branches and the conflicts has probably solved wrongly.

Can we just remove this commit ?

Reply:
Okay, earlier this morning i was preparing 3.2.0-rc (mistake, the date have been moved to later). So i changed this file not in the right timing. I just changed back the file on staging.

Reply:
Just added one commit here to be fully equal to staging, i think now @escopecz we could consider this a resolved ?

Reply:
reverted